private_gpt/ui/ui.py

```diff
@@ -271,12 +275,13 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
-                with gr.Column(scale=7):
+                with gr.Column(scale=7, elem_id='col'):
```

Comment:
This will make the formatter check pass. 

```suggestion
                with gr.Column(scale=7, elem_id="col"):
```

Anyway, make sure you run `make check` locally to run the formatting fixes and commit any pending fix.